src/components/Header/Header.tsx

```diff
@@ -1,15 +1,65 @@
 import { Button, Typography } from '@mui/material';
 import * as Styled from './Header.styles';
+import { useNavigate } from 'react-router-dom';
+import { isSignedIn, signOut, getUserEmail, onSignChange } from '@utils/login';
+import { render } from 'react-dom';
+import { useState } from 'react';
 
 const Header = () => {
+  const navigate = useNavigate();
+  const [signedIn, setSignedIn] = useState(isSignedIn());
```

Comment:
Make it isSignedIn - for booleans always try to start with 'is' - it's more readable that way.

Reply:
Addressed in commit: [03298211be1510af94c17aaecbe74406efdebccf](https://github.com/Luvkitri/zpab-frontend/pull/2/commits/03298211be1510af94c17aaecbe74406efdebccf)